### ISSUE TITLE
Metadata has no value in notifications

### ISSUE THREAD

**Issue by ctc-nakatsuka** [Fiware/context.Orion-LD#854]
When an entity sent in a notification contains metadata, the value property of the metadata may not be present, as shown below.
This seems to occur when the metadata value is an object or array.

```
{
  "id": "urn:ngsi-ld:Notification:60acbbebdb256bf6b027c3e3",
  "type": "Notification",
  "subscriptionId": "urn:ngsi-ld:Subscription:60acb735db256bf6b027c3df",
  "notifiedAt": "2021-05-25T08:57:15.429Z",
  "data": [
    {
      "id": "urn:ngsi-ld:Building:store001",
      "type": "Building",
      "name": {
        "type": "Property",
        "value": "Example",
        "foo": {
          "type": "Property",
          "": ["Example"]                  <-- here
        }
      }
    }
  ]
}
```

It occurs in the following version:

```
{
  "orionld version": "0.7.0",
  "orion version":   "1.15.0-next",
  "uptime":          "0 d, 0 h, 25 m, 13 s",
  "git_hash":        "nogitversion",
  "compile_time":    "Fri Mar 19 12:26:12 UTC 2021",
  "compiled_by":     "root",
  "compiled_in":     "buildkitsandbox",
  "release_date":    "Fri Mar 19 12:26:12 UTC 2021",
  "doc":             "https://fiware-orion.readthedocs.org/en/master/"
}
```

**Comment by kzangeli** [comment]
ok!
Interesting.
Definitely looks like a bug

**Comment by kzangeli** [comment]
This issue should be fixed by  #c9da921

**Comment by ctc-nakatsuka** [comment]
It works. Thank you!
